react/ShippingHeader.tsx

```diff
@@ -3,17 +3,26 @@ import { ButtonPlain } from 'vtex.styleguide'
 import { PlaceDetails } from 'vtex.place-components'
 import { FormattedMessage } from 'react-intl'
 import { AddressContext } from 'vtex.address-context'
+import { OrderForm } from 'vtex.order-manager'
 
 interface Props {
   onEditAddress?: () => void
 }
 
 const { useAddressContext } = AddressContext
+const { useOrderForm } = OrderForm
 
 const ShippingHeader: React.VFC<Props> = ({ onEditAddress }) => {
   const { address, invalidFields } = useAddressContext()
+  const {
+    orderForm: { canEditData },
+  } = useOrderForm()
 
-  if (!address || invalidFields.includes('postalCode')) {
+  if (
+    !address ||
+    ((canEditData || address?.isDisposable) &&
+      invalidFields.includes('postalCode'))
```

Comment:
question: when/how does `canEditData` changes?

Reply:
I think there is only a few scenarios:

- first purchase; `canEditData: true`
- second purchase, not logged in: `canEditData: false`
- second purchase, logged in: `canEditData: true`